src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -31,9 +31,10 @@ const ResendView = BaseResendView.extend(
       this.showCalloutAfterTimeout();
     },
 
-    handleResendLink() {
+    handleResendLink(ev) {
       this.options.appState.trigger('invokeAction', this.options.resendEmailAction);
       this.hideResendViewAndShowAfterTimeout();
+      ev?.preventDefault();
```

Comment:
keep this at top of function? in case unexpected future changes happen above this line

Reply:
Done